From 3c387371d7813c7fed338b9492b1933a3c9417ac Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Tue, 4 Aug 2020 11:14:06 -0700
Subject: [PATCH 1508/1852] dt-bindings: display: xlnx: mixer: Updated layer
 max height and width ranges

commit ccdf50b979b26fd97138301797f1b322ec9c38d5 from
https://github.com/Xilinx/linux-xlnx.git

Added layer maximum height and width range.

Signed-off-by: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
index a0b2fc05d5aa..06a5bad2a183 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
@@ -29,9 +29,9 @@ Required properties:
  - xlnx,vformat: video format for layer. See list of supported formats below.
  - xlnx,layer-max-width: max layer width, mandatory for master layer
    for overlay layers if scaling is alowed then this is mandatory otherwise
-   not required for overlay layers
+   not required for overlay layers. Valid range is 64 to 8192
  - xlnx,layer-max-height: max layer height, mandatory for master layer
-   Not required for overlay layers
+   Not required for overlay layers. Valid range is 64 to 4320
  - xlnx,layer-primary: denotes the primary layer, should be mentioned in node
    of layer which is expected to be constructing the primary plane
 
-- 
2.31.1

